ksy_schema.json

```
@@ -263,7 +296,13 @@
                 "cases"
               ]
             }
-          ]
+          ],
+          "description": "specifies the default byte order used in all fields of type `sX` or `uX` of the defining type and all it's subtypes\n\nthis field is required if you use types `sX` or `uX` without explicitly defined endianness"
```

```suggestion
          "description": "specifies the default byte order used in all fields of type `sX` or `uX` of the defining type and all its subtypes\n\nthis field is required if you use types `sX` or `uX` without explicitly defined endianness"
```

- ~_it's_~ = it is
- **its** - possessive pronoun of _it_ (something belongs to _it_)

See https://www.grammarly.com/blog/possessive-pronouns/: _A Common Mistake: Its vs. It’s_